### PR TITLE
PLUGIN-1613 : adding ignoreEmptySplits = false as spark 3.2 onwards t…

### PR DESCRIPTION
**Pull request by sahusanket**
…here was a behavior change with database plugins

### CODE REVIEW

File: cdap-app-templates/cdap-etl/hydrator-spark-core-base/src/main/java/io/cdap/cdap/etl/spark/batch/ETLSpark.java [hunk 1]
```diff
@@ -109,6 +109,9 @@ public void initialize() throws Exception {
     // to make sure fields that are the same but different casing are treated as different fields in auto-joins
     // see CDAP-17024
     sparkConf.set("spark.sql.caseSensitive", "true");
+    //For spark 3.2 onwards the spark.hadoopRDD.ignoreEmptySplits is set to true, which created ZERO stages while using
+    //database plugins : see PLUGIN-1613
+    sparkConf.set("spark.hadoopRDD.ignoreEmptySplits", "false");
```
Comment (iam-divya) [inline cdap-app-templates/cdap-etl/hydrator-spark-core-base/src/main/java/io/cdap/cdap/etl/spark/batch/ETLSpark.java:114]: Do we need similar fix in replication to get the same drivers working there? @sumitjnn

**Review by iam-divya — APPROVED** [review]
LGTM

File: cdap-app-templates/cdap-etl/hydrator-spark-core-base/src/main/java/io/cdap/cdap/etl/spark/batch/ETLSpark.java (hunk 1 above)
Comment (sahusanket) [reply cdap-app-templates/cdap-etl/hydrator-spark-core-base/src/main/java/io/cdap/cdap/etl/spark/batch/ETLSpark.java:114]: Replication doesn't use `spark.hadoopRDD` , so Replication should not be related to this change.

Comment (iam-divya) [reply cdap-app-templates/cdap-etl/hydrator-spark-core-base/src/main/java/io/cdap/cdap/etl/spark/batch/ETLSpark.java:114]: Got it.

**Pull request merged**